operator/v1/types_ingress.go

```diff
@@ -716,8 +720,58 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	// eipAllocations cannot be used with "Internal" scope.
+	// An EIP can be allocated to only a single IngressController.
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
+	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
+	// +optional
+	// +listType=atomic
+	// Each entry in the eipAllocations list must be unique
```

Comment:
Make sure all docs come before the markers please

Reply:
yes done.